packages/wonder-blocks-icon-button/src/themes/default.ts

```diff
@@ -145,14 +145,22 @@ const theme = {
                 ...baseColorStates.progressive,
                 default: {
                     ...baseColorStates.progressive.default,
-                    foreground: semanticColor.text.secondary,
+                    foreground: semanticColor.icon.primary,
+                },
+                hover: {
+                    ...baseColorStates.progressive.default,
```

Comment:
Should this be `baseColorStates.progressive.hover` since this is for the hover styles?

Reply:
Good catch! 😅